src/index.ts

```diff
@@ -66,6 +66,10 @@ function nextTranslate(nextConfig: NextConfig = {}): NextConfig {
     return nextConfigWithI18n
   }
 
+  if (appDirNext13.includes(pagesInDir)) {
+    isAppDirNext13 = true
+  }
```

Comment:
maybe make sense to **remove** the line **52** and **58** and replace this to:
```suggestion
const isAppDirNext13 = appDirNext13.includes(pagesInDir)
```

Reply:
Yup, definitely! I will update that shortly :-)